src/User32/User32+MENUINFO.cs

```diff
@@ -49,7 +49,7 @@ public struct MENUINFO
             /// Create a new instance of <see cref="MENUINFO"/> with <see cref="cbSize"/> set to the correct value.
             /// </summary>
             /// <returns>A new instance of <see cref="MENUINFO"/> with <see cref="cbSize"/> set to the correct value.</returns>
-            public MENUINFO Create() => new MENUINFO { cbSize = Marshal.SizeOf(typeof(MENUINFO)) };
+            public static unsafe MENUINFO Create() => new MENUINFO { cbSize = sizeof(MENUINFO) };
```

Comment:
This `Create()` method wasn't `static`, which was another bug that I fixed along the way.